src/drivers/distance_sensor/tfmini/tfmini_parser.cpp

```diff
@@ -132,20 +132,20 @@ int tfmini_parse(char c, char *parserbuf, unsigned *parserbuf_index, TFMINI_PARS
 		break;
 
 	case TFMINI_PARSE_STATE::STATE3_GOT_STRENGTH_H:
-		*state = TFMINI_PARSE_STATE::STATE4_GOT_RESERVED;
+		*state = TFMINI_PARSE_STATE::STATE4_GOT_CHIP_TEMPL;
 		parserbuf[*parserbuf_index] = c;
 		(*parserbuf_index)++;
 
 		break;
 
-	case TFMINI_PARSE_STATE::STATE4_GOT_RESERVED:
-		*state = TFMINI_PARSE_STATE::STATE5_GOT_QUALITY;
+	case TFMINI_PARSE_STATE::STATE4_GOT_CHIP_TEMPL:
+		*state = TFMINI_PARSE_STATE::STATE5_GOT_CHIP_TEMPH;
 		parserbuf[*parserbuf_index] = c;
 		(*parserbuf_index)++;
 
 		break;
 
-	case TFMINI_PARSE_STATE::STATE5_GOT_QUALITY:
```

Comment:
Is there any quality metric we can get from the newer TF lidar?

Reply:
Ok, I think I should keep separate copies for each model, this will make things crystal clear. Am I allowed to edit this pull request and resubmit it again or I should open a new pull request and close this?